endoreg_db/services/ollama_api_docs.py

```
@@ -0,0 +1,1528 @@
+API reference
+​
```

Pylint (4.0.0)

[error] Parsing failed: 'invalid non-printable character U+200B (endoreg_db.services.ollama_api_docs, line 2)'

(E0001)